library/n2_protocol.hpp

```diff
@@ -73,7 +103,7 @@ struct read_response : n2_message {
 	uint64_t read_data_size;
 	data_place data;
 
-	virtual void make_owning() override { /*TODO(sabramkin): implement*/ }
+	virtual void make_owning() override;
```

Comment:
`override` is enough.

Reply:
Still be not fixed.
https://google.github.io/styleguide/cppguide.html#Inheritance

> Do not use virtual when declaring an override.

Reply:
Ok, thanks. After first your comment I didn't understand what did you mean.